aws/resource_aws_route53_zone_test.go

```diff
@@ -222,30 +239,9 @@ func TestAccAWSRoute53Zone_ForceDestroy(t *testing.T) {
 	})
 }
 
-func TestAccAWSRoute53Zone_ForceDestroy_TrailingPeriod(t *testing.T) {
-	var zone route53.GetHostedZoneOutput
-
-	rString := acctest.RandString(8)
-	resourceName := "aws_route53_zone.test"
-	zoneName := fmt.Sprintf("%s.terraformtest.com", rString)
-
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckRoute53ZoneDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccRoute53ZoneConfigForceDestroyTrailingPeriod(zoneName),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckRoute53ZoneExists(resourceName, &zone),
-					// Add >100 records to verify pagination works ok
-					testAccCreateRandomRoute53RecordsInZoneId(&zone, 100),
-					testAccCreateRandomRoute53RecordsInZoneId(&zone, 5),
-				),
-			},
-		},
-	})
-}
+// TestAccAWSRoute53Zone_ForceDestroy_TrailingPeriod removed in v3.0.0
```

Comment:
Drive-by comment (I know this is still WIP 😅 ): since we're still allowing the trailing period in the argument, I think we can keep this old test case, just to be sure the strange behavior we saw previously with pagination is still covered and working as expected. The Route53 API be full of dragons. 👍

Reply:
good to know! i may have gone about removing these types of tests in the other PRs so I'll double-check. b245da3 re-adds it here